content/en/docs/userguide/install_waypoint.md

```diff
@@ -76,50 +80,54 @@ Label the `reviews` service to use the `reviews-svc-waypoint` waypoint:
+
+You can use `kubectl get pods` to see all the pods except waypoint are ready. **Then replace the waypoint image with the Kmesh customized image.** Based on istio-proxy, Kmesh adds an customized listener filter called [Kmesh_tlv](https://github.com/kmesh-net/waypoint/tree/master/source/extensions/filters/listener/kmesh_tlv), which will parse the custom TLV protocol encoded by Kmesh and obtain the target address and metadata to connect L4 and L7.
 
 ```bash
 [root@ ~]# kubectl get gateways.gateway.networking.k8s.io
 NAME                      CLASS            ADDRESS        PROGRAMMED   AGE
 reviews-svc-waypoint      istio-waypoint   10.96.198.98   True         30m
 ```
 
-Add annotation "sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint:latest" to the `bookinfo-reviews` gateway.
+**image replacement**: Add annotation "sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint:latest" to the `reviews-svc-waypoint` gateway.
 
 ```bash
 [root@ ~]# kubectl annotate gateway reviews-svc-waypoint sidecar.istio.io/proxyImage=ghcr.io/kmesh-net/waypoint:latest
 ```
 
 Then gateway pod will restart. Now Kmesh is L7 enabled!
 
-In addition, you can also use waypoint at namespace or pod granularity.
+```bash
+[root@ ~]# kubectl get pods
+NAME                                      READY   STATUS    RESTARTS   AGE
+details-v1-cdd874bc9-xcdnj                1/1     Running   0          30m
+productpage-v1-5bb9985d4d-z8cws           1/1     Running   0          30m
+ratings-v1-6484d64bbc-pkv6h               1/1     Running   0          30m
+reviews-svc-waypoint-8cb4bdbf-9d5mj       1/1     Running   0          30m
+reviews-v1-598f9b58fc-2rw7r               1/1     Running   0          30m
+reviews-v2-5979c6fc9c-72bst               1/1     Running   0          30m
+reviews-v3-7bbb5b9cf7-952d8               1/1     Running   0          30m
+sleep-5577c64d7c-n7rxp                    1/1     Running   0          30m
+```
+
 
-#### Use waypoint at namespace granularity:
+#### Install waypoint in namespace granularity:
 
 ```bash
-[root@ ~]# istioctl x waypoint apply -n default --enroll-namespace
-waypoint default/waypoint applied
-namespace default labeled with "istio.io/use-waypoint: waypoint"
+[root@ ~]#  istioctl x waypoint apply -n default --name default-ns-waypoint
+waypoint default/default-ns-waypoint applied
+
+[root@ ~]#  kubectl label namespace default istio.io/use-waypoint=default-ns-waypoint
+namespace/default labeled
 ```
 
 Then any requests from any pods using the Kmesh, to any service running in `default` namespace, will be routed through that waypoint for L7 processing and policy enforcement.
 
-#### Use waypoint at pod granularity:
+#### Install waypoint in pod granularity:
 
 ```bash
-[root@ ~]# istioctl x waypoint apply -n default --name reviews-v2-pod-waypoint --for workload
+[root@ ~]# istioctl x waypoint apply -n default --name reviews-v2-pod-waypoint
```

Comment:
This is wrong ,change it back. 

For pod waypoint, we need the suffix of `--for workload`